core/src/main/java/dk/alexandra/fresco/framework/sce/evaluator/BatchedStrategy.java

```diff
@@ -58,34 +53,24 @@
  *
  * The processing is done is in a sequential manner (i.e. no parallelization).
  */
-public class BatchedStrategy {
+public class BatchedStrategy<ResourcePoolT extends ResourcePool> implements BatchEvaluationStrategy<ResourcePoolT>{
 
-  /**
-   * @param protocols array holding the protocols to be evaluated
-   * @param sceNetwork array of sceNetworks corresponding to the protocols to be evaluated.
-   *     I.e.,
-   *     the array should contain numProtocols SCENetworks, with sceNetwork[i] used for
-   *     communication in
-   *     protocols[i].
-   * @param channel string indicating the channel to communicate over.
-   * @param rp the resource pool.
-   */
-  public static <ResourcePoolT extends ResourcePool> void processBatch(
-      ProtocolCollection<ResourcePoolT> protocols,
-      SCENetwork sceNetwork, int channel, ResourcePoolT rp) throws IOException {
-    Network network = rp.getNetwork();
+  @Override
+  public <sceNetwork extends SCENetwork & SCENetworkSupplier> void processBatch(
+      ProtocolCollection<ResourcePoolT> protocols, ResourcePoolT resourcePool, sceNetwork sceNetwork)
+          throws IOException {
+    Network network = resourcePool.getNetwork();
     int round = 0;
-
     while (protocols.size() > 0) {
-      evaluateCurrentRound(protocols, sceNetwork, channel, rp, network, round);
+      evaluateCurrentRound(protocols, sceNetwork, 0, resourcePool, network, round);
```

Comment:
I agree - channel is obsolete - why not remove it from the interface?

Reply:
Mostly because we would like in theory to play around with the idea of multiple channels - this enables threading such that a batch can be split into multiple worker threads - each with their own channel. 
This quickly becomes complicated, and was the reason we pulled it from the framework. Still - it might come in handy at some point

Reply:
Ideas are free, code is not.

I would suggest to keep the idea but remove these legacy elements from the actual code making the future expansion possible without regard to the old version.

There are several ways of expanding the throughput over channels, the now removed implementation was not very good.